include/dr_param/yaml.hpp

```diff
@@ -109,11 +109,21 @@ estd::result<YAML::Node, estd::error> readYamlFile(std::string const & path);
 /// Merge the nodes of map_b into map_a and overwrite if it already exists in map_a.
 YamlResult<void> mergeYamlNodes(YAML::Node & map_a, YAML::Node map_b);
 
-/// Merge the nodes of map_b into map_a and overwrite if it already exists in map_a.
-inline YamlResult<void> mergeYamlNodes(YAML::Node && map_a, YAML::Node map_b) {
-	return mergeYamlNodes(map_a, map_b);
+/// Merge Yaml Nodes of type Ordered Dictionary, i.e., sequence on Maps.
+YamlResult<void> mergeYamlOrderedDict(YAML::Node & map_a, YAML::Node map_b);
```

Comment:
I don't think these new functions should be in the public header. We only need them to implement `mergeYamlNodes`, so the public API should be unaffected. Could you remove the new declarations and restore the old one that was removed?